locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/networking/JcloudsLocationSecurityGroupCustomizer.java

```diff
@@ -127,6 +127,7 @@ protected JcloudsLocationSecurityGroupCustomizer(String applicationId) {
     }
 
     protected JcloudsLocationSecurityGroupCustomizer(String applicationId, Supplier<Cidr> sshCidrSupplier) {
+        super();
```

Comment:
I find myself adding `super()` at times (almost unconsciously!) as a reminder-to-self when I happen to be thinking about inherited behaviour.  It's probably a bad habit!  I'll take this out again, it can be helpful for breakpoints as you say but I don't think there's any need for that here.